Clients/src/Swag4Net.Core/v3/SpecificationDocument.fs

```diff
@@ -0,0 +1,245 @@
+namespace Swag4Net.Core.v3
+
+module SpecificationDocument =
+
+  type TypeName = string
+  type HttpStatusCode = string
+  type TypeOrReference<'a> =
+     | T of 'a
```

Comment:
Please use more explicit names

Reply:
you mean for the type parameter for TypeOrReference? That's right, i change that